lib/dotetch/readme.md

```diff
@@ -0,0 +1,24 @@
+# .Etch
+
+## Genration
+
+Entry is `dotEtchStream.ts`
```

Comment:
dotetch/index.ts ?

Reply:
indeed that has been rewrote

Reply:
Actually that readme was quite outdated, this is now fixed